database/migrations/2018_06_04_125200_create_process_users_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateProcessUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('PROCESS_USER', function(Blueprint $table)
+        {
+            $table->uuid('PU_UID')->default('')->primary();
```

Comment:
Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.